app/services/call_creator.rb

```diff
@@ -46,10 +46,16 @@ def errors
   private
 
   def sanitize_params!
-    if @params[:member_phone_number].present?
-      @params[:member_phone_number] = Phony.normalize(@params[:member_phone_number])
+    begin
+      if @params[:member_phone_number].present?
+        @params[:member_phone_number] = Phony.normalize(@params[:member_phone_number])
+      end
+    rescue Phony::NormalizationError
+    end
+
+    if @params[:member_id].blank? && @extra_params[:akid].present?
+      @params[:member_id] = Member.find_by_akid(@extra_params[:akid])&.id
```

Comment:
Hmmm ok. It would be more internally consistent if we had all of our plugins submit the params in the same format, so that we could just do all our member recognition in one place, but I'm happy to merge this to fix the current bug.